src/cmd/download-tachyon-package.js

```diff
@@ -46,7 +46,7 @@ module.exports = class DownloadTachyonPackageCommand extends CLICommandBase {
 		}
 		const manager = new DownloadManager(this.ui);
 		const manifest = await manager.fetchManifest({ version });
-		const build = manifest.builds.find((b) => b.region === region && b.variant === variant);
+		const build = manifest.builds.find((b) => b.region === region && b.variant === variant && b.board === board);
 		if (!build) {
 			throw new Error(`No build available for region: ${region}`);
```

Comment:
The message no build available shows up a couple of times. I think it's sufficient to say "No build available for the provided parameters" without listing them all out.

Reply:
Fixed here! https://github.com/particle-iot/particle-cli/pull/803/commits/1b2353b3bf37865fbd50c66ff21390256911a6d3